http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -205,6 +205,10 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         s"Relay listener values for app ${params.appName.value} are ${values.asString}"
       )
 
+      // Pull Listener helm chart
+      _ = helmClient
```

Comment:
I think this would need to be 
```
_ <- helmClient.updateAndPullChart(config.listenerChartConfig.chartName, config.listenerChartConfig.chartVersion, "/leonardo").run(authContext)
```

Since it's an `F` effect, it probably is not being run if you just `_ = helmClient.updateAndPullChart(..)`.

Reply:
Hm hm - maybe this?
```
_ <- helmClient.updateAndPullChart(config.listenerChartConfig.chartName, config.listenerChartConfig.chartVersion, "/leonardo").run(())
```

Since it's defined as a `Kleisli[F, Unit, Unit]`:
https://github.com/broadinstitute/helm-scala-sdk/blob/master/src/main/scala/org/broadinstitute/dsp/HelmAlgebra.scala#L20

(It probably doesn't need to be a `Kleisli` at all, which is where the `.run` comes from..

Reply:
Ahhh, I had tried `.run()`, but not `run(())` 😓